src/pj_param.c

```diff
@@ -35,76 +93,76 @@ paralist *pj_mkparam(char *str) {
 /*                                                                      */
 /************************************************************************/
 
-	PROJVALUE /* test for presence or get parameter value */
+    PROJVALUE /* test for presence or get parameter value */
```

Comment:
If the union is char *s, who owns the string?

Reply:
This is classic 1980s PROJ code, which I consider eliminating (the `pj_param_exists` function is the first step towards this). I see your point in being clear about memory deallocation duty, and I have spent a lot of time adding comments to older PROJ code, but in this case, I will really prefer to work toward better solution, rather than documenting code with more complex use patterns than need be.